pubspec.yaml

```diff
@@ -40,9 +40,9 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  build_runner: ^2.1.2
+  build_runner: ^2.1.7
   freezed: ^0.14.5
-  json_serializable: ^5.0.0
+  json_serializable: ^6.1.4
```

Comment:
As json serializable package is upgraded to next major version, please run build runner command again to update `*.g.dart` files also

Reply:
@namanpopli